### ISSUE TITLE
[BUG] `fd -h` panics when stdout is closed

### ISSUE THREAD

**Issue by lopopolo** [sharkdp/fd#897]
```console
$ RUST_BACKTRACE=1 fd -h | false
thread 'main' panicked at 'Error writing Error to stdout: Os { code: 32, kind: BrokenPipe, message: "Broken pipe" }', /Users/brew/Library/Caches/Homebrew/cargo_cache/registry/src/github.com-1ecc6299db9ec823/clap-2.33.3/src/errors.rs:401:55
stack backtrace:
   0: _rust_begin_unwind
   1: core::panicking::panic_fmt
   2: core::result::unwrap_failed
   3: core::result::Result<T,E>::expect
   4: clap::errors::Error::exit
   5: clap::app::App::get_matches_from::{{closure}}
   6: fd::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

See related:

- https://github.com/sharkdp/hyperfine/issues/453

**Comment by lopopolo** [comment]
I've opened up this ticket with upstream clap to see if the maintainers are open to fixing this panic on the v2 branch:

- https://github.com/clap-rs/clap/discussions/3055

**Comment by sharkdp** [comment]
closed via #900
